src/main/java/org/team1540/robot2025/subsystems/elevator/ElevatorIO.java

```diff
@@ -25,7 +23,11 @@ default void updateInputs(ElevatorIOInputs inputs) {}
 
     default void setVoltage(double volts) {}
 
-    default void setSetpointMeters(double setpoint) {}
+    default void setSetpoint(double setpointMeters) {}
 
     default void setBrakeMode(boolean brakeMode) {}
+
+    default void configPID(double kP, double kI, double kD) {}
+
+    default void configFF(double kS, double kV, double kA) {}
```

Comment:
`kA` isn't often useful, it's probably more helpful to have `kG` for an elevator